src/adams_methods.jl

```diff
@@ -0,0 +1,655 @@
+################################################################################
+# Fixed Step Adams-Bashforth multi-step solver for nonstiff problems
+#
+#     [T,X] = ode_ms(ODEFUN, Y0, TSPAN,ORDER=4) with TSPAN = [T0:H:TFINAL]
+#
+#     Note: If method is run at order k, the first step is taken at order 1,
+#     and the order increases each step until it is order k on the kth step.
+#     Thus results may be of lower accuracy than expected.
+#     (Main ref: Hairer & Wanner 1996, Vol I, p.359-360, 372)
+################################################################################
+
+function ode_ms(F, y0, tspan, order::Integer)
```

Comment:
I think this solver only works when all `h` are equal, correct?  If so you should add test & error for that.